### DISCUSSION TITLE
Git repositories images are not loading

### DISCUSSION THREAD

**Discussion by kiishanbarmawala** [community/community#51504] [Repositories]
### Select Topic Area

Bug

### Body

I Have found that, from last 4-5 hours, images are not loading/showing on not only mine, but any of the git repos images also are not showing. Can you please check the same?

**Comment by leanczo** [comment]
Sorry to reply late, but today that did not happen to me, and now I see that they work well for me. Greetings

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @kishanbarmawala ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
